Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook

```diff
@@ -67,6 +88,44 @@
       },
       "name": "overview_section_params"
     },
+    {
+      "type": 11,
```

Comment:
I think we should make this button conditionally visible like only when tables are created in LA or Provider creation is successful. You can use the parameters - param_check_provider_error, and param_check_error to check that.

Reply:
Made use of these parameters for conditional visibility of Alerts button